app/controllers/certs_controller.rb

```diff
@@ -10,8 +10,10 @@ def show
     @last_editor = last_editor(@cert)
   end
 
- def new
-    @cert.person_id = (certs_params[:person_id]) if (certs_params[:person_id]).present?
+  def new
+    if params[:cert]
```

Comment:
Is this needed ? I thought the `if (certs_params[:person_id]).present?` at the end would cover this ?